media_driver/linux/common/os/i915/mos_bufmgr.c

```diff
@@ -1700,6 +1886,22 @@ drm_export int mos_gem_bo_map(struct mos_linux_bo *bo, int write_enable)
         }
         VG(VALGRIND_MALLOCLIKE_BLOCK(mmap_arg.addr_ptr, mmap_arg.size, 0, 1));
         bo_gem->mem_virtual = (void *)(uintptr_t) mmap_arg.addr_ptr;
+
+        memclear(set_domain);
```

Comment:
Another difference vs. i915_production/mos_bufmgr.c.

See https://github.com/freedesktop/drm-tip/blob/85801dbe86b9dc8eae00dd10347b7f3552f9bef9/include/uapi/drm/i915_drm.h#L939:
```
 * Note that for discrete, starting from DG1, this is no longer supported, and
 * is instead rejected. On such platforms the CPU domain is effectively static,
 ...
```

In other words, we get -1 calling DRM_IOCTL_I915_GEM_SET_DOMAIN on DG1.